src/App.tsx

```diff
@@ -1,27 +1,65 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Button from 'react-bootstrap/Button';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [allGoods, setAllGoods] = useState<Good[]>([]);
+  const [isSelected, setIsSelected] = useState(false);
```

Comment:
why all 3 buttons are saying `Loading` when I clock just on one of them?
![image](https://user-images.githubusercontent.com/37161315/228550610-612fd48c-0f0c-4eae-8750-e5fd66f25d49.png)
Let's instead show some fancy loader